datafusion/physical-optimizer/Cargo.toml

```diff
@@ -41,6 +41,7 @@ datafusion-common = { workspace = true, default-features = true }
 datafusion-execution = { workspace = true }
 datafusion-expr = { workspace = true }
 datafusion-expr-common = { workspace = true, default-features = true }
+datafusion-functions-aggregate = { workspace = true }
```

Comment:
I would like to avoid this dependency if possible -- I will like to make a follow on PR to remove it -- similar to how I did in https://github.com/apache/datafusion/pull/14134

However when I just used the stub a few tests started failing on me